app/lib/submission_builder/ty2024/states/md/documents/md502.rb

```diff
@@ -72,8 +72,8 @@ def document
           xml.send(filing_status, "X")
         end
       end
-      if has_exemptions?
-        xml.Exemptions do
+      xml.Exemptions do
+        if has_exemptions?
```

Comment:
why did you flip this? in the XSD it appears that this element is not required.

Reply:
If you look at the convo in the Jira the issue is that the XSD is not fully accurate. We need field A and D filled always.

https://codeforamerica.atlassian.net/browse/FYST-1717